pkg/apis/kudo/v1beta1/operatorversion_types_helpers.go

```diff
@@ -32,3 +45,27 @@ func GetOperatorVersionByName(name, ns string, c client.Reader) (ov *OperatorVer
 	}
 	return ov, nil
 }
+
+// Sortable Operator implements functionality to correctly sort OVs by name, appVersion and operatorVersion
+var _ kudo.SortableOperator = &OperatorVersion{}
```

Comment:
What's the purpose of blank assignment? To indicate that `OperatorVersion` is supposed to implement the `kudo.SortableOperator` interface?

Reply:
Yes - it's basically an assert that `OperatorVersion` implements `kudo.SortableOperator` - So if somebody adds a new func to `SortableOperator` we get a compile time error and not a runtime error some point later.

Reply:
Nice!